src/toast-notification/classes.scss

```diff
@@ -60,13 +55,50 @@
 
     .iui-toast-entering,
     .iui-toast-entered {
-      animation: openAnimation 240ms cubic-bezier(0.175, 0.885, 0.32, 1.175) both;
+      animation: openAnimation 240ms cubic-bezier(0.175, 0.885, 0.32, 1.175)
+        both;
     }
 
     .iui-toast-exiting {
       animation: closeAnimation 120ms cubic-bezier(0.4, 0, 1, 1) both;
     }
   }
+
+  &.iui-placement-top {
+    margin-left: auto;
+    margin-right: auto;
+    left: 0;
+    top: 0;
+    right: 0;
```

Comment:
Do you really need left:0 and right:0 here?

Reply:
Same thing in iui-placement-bottom

Reply:
Without these positions toaster is placed on the left side. We need left:0, right:0.